charts/keycloak/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: keycloak
-version: 10.1.0
+version: 10.2.0
```

Comment:
Another PR was merged in between and you have to increase the version number once again... Sorry for that

Reply:
No problem, I've incorporated your feedback!